src/profile/v100/entity/persona/persona_data/collection_of_email_addresses.rs

```diff
@@ -0,0 +1,23 @@
+use crate::prelude::*;
+
+declare_collection_of_identified_entry!(
```

Comment:
refactored from file declaring the macro `declare_collection_of_identified_entry`